tests/broadcaster_test.py

```diff
@@ -98,11 +98,15 @@ def server(postgres):
     # Run the server as a separate process
     proc = Process(target=setup_server, args=(postgres, ), daemon=True)
     proc.start()
-    logger.info("Server started on a deamon process")
+    logger.info("Server started on a daemon process")
     yield proc
     proc.kill()  # Cleanup after test
 
 
+
+skip_unless_requested = pytest.mark.skipif(os.environ.get('TEST_BROADCAST') is None, reason="Not configured to test for broadcast (requires Postgres) | enable with TEST_BROADCAST=1")
```

Comment:
@orweis Why does broadcast test fails / becomes redundant when testing on Python3.10 ?
Or is it a completely different fix?